docs/quickstart/custom_dataset.md

```diff
@@ -108,6 +123,57 @@ cd vcpkg
 :::::
 ::::::
 
+(polycam)=
+
+## Polycam Capture
+
+Nerfstudio can also be trained directly from captures from the [Polycam app](https://poly.cam//). This avoids the need to use COLMAP.
+
+:::{admonition} Note
+:class: info
+A LiDAR enabled iPhone or iPad is necessary.
+:::
+
+### Setting up Polycam
+
+```{image} imgs/polycam_settings.png
+:width: 200
+:align: center
+:alt: polycam settings
+```
+
+Devoloper settings must be enabled in Polycam. To do this, navigate to the settings screen and select `Developer mode`. Note that this will only apply for future captures, you will not be able to process existing captures with nerfstudio.
+
+### Process data
+
+```{image} imgs/polycam_export.png
+:width: 400
+:align: center
+:alt: polycam export options
+```
+
+0. Capture data in LiDAR or Room mode.
+
+1. Tap `Process` to process the data in the Polycam app.
+
+2. Navigate to the export app pane.
+
+3. Select `raw data` to export a `.zip` file.
+
+4. Convert the Polycam data into the nerfstudio format using the following command:
+
+```bash
+ns-process-data polycam --data {OUTPUT_FILE.zip} --output-dir {output directory}
+```
+
+5. Train the model using the following command:
+
+```bash
+ns-train nerfacto --data {output directory}
+```
+
```

Comment:
It might be helpful to add some variant of these data collection tips: https://github.com/PolyCam/polyform#tips-on-dataset-collection to the readme 

Blurry images are the enemy of 3D reconstruction, and it's easy to get these when scanning in video mode since the camera is always moving, so I always suggest using the manual keyframer if the lighting is not great.

You might also make a note that Polycam's poses are globally optimized or else point them to our readme if they want more info about our data. This might seem a bit pedantic, but anyone who has worked with ARKit or SLAM poses in general know that drift is a big issue, so calling that out might help to build more trust in this data source.

Reply:
This is a great point! What is the maximum number of frames during bundle adjustment?

Reply:
By default we bundle adjust up to 700 frames. Although users can enable loop closure via the CUSTOM processing panel for scans up to 1400 frames. 
![Screen Shot 2022-11-15 at 3 53 50 PM](https://user-images.githubusercontent.com/12475454/202049373-be747715-6b3e-4c17-93f4-6b78073eb815.png)